setup.cfg

```diff
@@ -23,28 +21,32 @@ classifiers =
     Topic :: Utilities
     Typing :: Typed
 keywords =
-    package
-    url
-    package manager
-    package url
+    Scorecard
+    ScoreCode
+    OpenSSF
+    OSSF
+    Vulnerablity Analysis
 license_files =
     LICENSE
     - AUTHORS.rst
     - README.rst
     - CONTRIBUTING.rst
     - CHANGELOG.rst
+    - apache-2.0.LICENSE
 
 [options]
-python_requires = >=3.8
```

Comment:
We should keep these options as I think they are needed to create wheels/source archives for a release